docs/Oracle Jobs/initiators.md

```diff
@@ -5,6 +5,15 @@ title: "Initiators"
 permalink: "docs/initiators/"
 whatsnext: {"Introduction to External Initiators":"/docs/external-initiators-introduction/"}
 ---
```

Comment:
Do we keep the External Initiators in V2? I mean not only here but all of the relevant pages/docs.